atomix/storage/src/test/java/io/atomix/storage/journal/AbstractJournalTest.java

```diff
@@ -90,8 +92,8 @@ public static Collection primeNumbers() {
         .withDirectory(folder)
         .withNamespace(NAMESPACE)
         .withStorageLevel(storageLevel())
-        .withMaxSegmentSize(maxSegmentSize)
-        .withMaxEntrySize(48)
+        .withMaxSegmentSize(maxSegmentSize * 2)
```

Comment:
Why is this required?

Reply:
The segment size is in the tests to small i was not able to write the max msg size😅

Reply:
maxSegmentSize is parameterized, right? May be we can adjust it to start from a higher value.

Reply:
Was not able to find it due to to the abstractions in the test classes. If you see it then feel free😃

Reply:
I think they are set somehow as arguments for the test classes